### PR TITLE
Initial draft of service accounts page

### PR DESCRIPTION
**Pull request by urseberry**
### What does this PR do?
Create a page to document service accounts

### Motivation
Product manager request

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/account_management/org_settings.md [hunk 1]
```diff
@@ -13,42 +13,42 @@ Read the [user management][2] documentation to add, edit, and disable users.
 
 #### Service accounts
 
-Service accounts are non-human users that can have roles and own application keys. You can use service accounts to access Datadog APIs without having your application or script be associated with a particular person. Service account application keys are one-time-view only, and are not retrievable after they've been created.
+[Service accounts][3] are non-human users that can have roles and own application keys. You can use service accounts to access Datadog APIs without having your application or script be associated with a particular person. Service account application keys are one-time-view only, and are not retrievable after they've been created.
```
Comment (louisthaboogie) [inline content/en/account_management/org_settings.md:16]: I would align the copy here with the same headline as the service accounts page. Sounds weird here that we call a service accounts a non human users. Maybe replace it with "Service accounts are non-interactive accounts that can .."

File: content/en/account_management/org_settings/service_accounts.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
+---
+title: Service Accounts
+kind: documentation
+---
+
+## Overview
+
+Service accounts are non-interactive accounts that can be used to own application keys and other resources that are shared across your teams. Service account application keys can only be viewed once, by the individual who created the key.
```
Comment (alai97) [inline content/en/account_management/org_settings/service_accounts.md:8]:
```suggestion
Service accounts are non-interactive accounts you can use to own application keys and other resources that are shared across your teams. Service account application keys can only be viewed once by the individual who created the key.
```

File: content/en/account_management/org_settings/service_accounts.md [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+- Permissions
+
+### Create service account
+
+To create a service account, perform the following steps:
+
+1. Click the **New Service Account** button on the service accounts page. A dialog box appears.
+2. In the pop-up dialog box, enter the name and email address for your new service account.
+3. Use the **Assign Roles** drop-down to choose one or more roles for your service account.
+4. To save, click **Create Service Account**.
```
Comment (alai97) [inline content/en/account_management/org_settings/service_accounts.md:43]:
```suggestion
1. Click **New Service Account**. A dialog box appears.
2. Enter a name and email address for your service account.
3. Use the **Assign Roles** drop-down menu to choose one or more roles for your service account.
4. To save, click **Create Service Account**.
```

File: content/en/account_management/org_settings/service_accounts.md [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+
+**Note:** Unlike the email addresses for Datadog users, service account email addresses do not need to be unique across an organization.
```
Comment (alai97) [inline content/en/account_management/org_settings/service_accounts.md:45]:
```suggestion
Unlike the email addresses for Datadog users, service account email addresses do not need to be unique across an organization.
```

File: content/en/account_management/org_settings/service_accounts.md [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+
+### Edit service account
+
+To modify a service account, click on it from the service accounts page.
```
Comment (alai97) [inline content/en/account_management/org_settings/service_accounts.md:49]:
```suggestion
To modify a service account, click on one in the service accounts list. 
```

File: content/en/account_management/org_settings/service_accounts.md [hunk 6]
```diff
@@ -0,0 +1,94 @@
...
+4. To save, click **Create Service Account**.
+
+**Note:** Unlike the email addresses for Datadog users, service account email addresses do not need to be unique across an organization.
+
+### Edit service account
+
+To modify a service account, click on it from the service accounts page. 
+1. In the side panel, click the **Edit** button next to the service account name. A dialog box appears.
+2. In the pop-up dialog box, update any fields you would like to change. You can edit the name, email address, status, and roles.
+3. Click **Save**.
```
Comment (alai97) [inline content/en/account_management/org_settings/service_accounts.md:52]:
```suggestion

1. In the side panel, click **Edit** next to the service account name. A dialog box appears.
2. Update any fields you would like to change. You can edit the name, email address, status, and roles.
3. Click **Save**.
```

File: content/en/account_management/org_settings/service_accounts.md [hunk 7]
```diff
@@ -0,0 +1,94 @@
...
+
+To disable a service account, follow the procedure above to edit the service account. In the pop-up dialog box, set the status to **Disabled**.
```
Comment (alai97) [inline content/en/account_management/org_settings/service_accounts.md:54]:
```suggestion
To disable a service account, follow the procedure above to edit the service account and set the status to **Disabled**.
```

File: content/en/account_management/org_settings/service_accounts.md [hunk 8]
```diff
@@ -0,0 +1,94 @@
...
+
+### Create or revoke application keys
+
+To create or revoke service account application keys, select the service account on the service accounts page. The service account detailed view side panel appears.
```
Comment (alai97) [inline content/en/account_management/org_settings/service_accounts.md:58]:
```suggestion
To create or revoke service account application keys, select an account from the service account list. The service account's side panel appears.
```

File: content/en/account_management/org_settings/service_accounts.md [hunk 9]
```diff
@@ -0,0 +1,94 @@
...
+
+To create a new application key, follow the steps below:
+- Click the **New Key** button. A dialog box appears.
+- Give the key a descriptive name.
+- Click **Create Key**.
```
Comment (alai97) [inline content/en/account_management/org_settings/service_accounts.md:63]:
```suggestion

- Click **New Key**. A dialog box appears.
- Give the key a descriptive name.
- Click **Create Key**. 
```

File: content/en/account_management/org_settings/service_accounts.md [hunk 10]
```diff
@@ -0,0 +1,94 @@
...
+To create or revoke service account application keys, select the service account on the service accounts page. The service account detailed view side panel appears.
+
+To create a new application key, follow the steps below:
+- Click the **New Key** button. A dialog box appears.
+- Give the key a descriptive name.
+- Click **Create Key**. 
+
+The dialog box refreshes, showing you the key. Copy and paste the key into your desired location. After you close the dialog box, you cannot retrieve the value of the key.
+
+To revoke an application key, find the key in the service account detailed view side panel. Mouse over it, and pencil and trash can icons appear on the right. Click the trash can to revoke the key. A dialog box pops up. Click **Confirm**.
```
Comment (alai97) [inline content/en/account_management/org_settings/service_accounts.md:67]:
```suggestion
To revoke an application key, find the key in the service account's side panel and hover over it. A pencil and trash can icon appears on the right. Click the trash can to revoke the key. After the key is revoked, click **Confirm**.
```

File: content/en/account_management/org_settings/service_accounts.md [hunk 11]
```diff
@@ -0,0 +1,94 @@
...
+
+See the [Service accounts API reference][2] to use service accounts through the Datadog API.
+
+## Service account application keys
+
+You can view a service account applications key exactly once, immediately after you create it. Limiting the access to the application key prevents any problems that may occur when another user accesses the key. If you lose or forget a service account key, revoke it and create a new one.
+
+## Permissions
+
+By creating a service account, you create an actor that interacts with Datadog on your behalf. Therefore, your capabilities on the service accounts page vary depending on your Datadog roles and permissions.
```
Comment (alai97) [inline content/en/account_management/org_settings/service_accounts.md:79]:
```suggestion
By creating a service account, you create an actor that interacts with Datadog on your behalf. Your capabilities on the Service Accounts page vary depending on your Datadog roles and permissions.
```

File: content/en/account_management/org_settings/service_accounts.md [hunk 12]
```diff
@@ -0,0 +1,94 @@
...
+
+Creating a service account requires the Service Account Write permission. The Datadog Admin role includes Service Account Write, so anyone with the Datadog Admin role can create service accounts.
+
+When you create a service account, you may give it any subset of the roles and permissions that you have. The exception is if you have the User Access Manage permission. Having User Access Manage effectively gives you administrator access to do anything in Datadog. Therefore, Datadog accounts with the User Access Manage permission have no restrictions on the roles and permissions they can assign to service accounts.
```
Comment (alai97) [inline content/en/account_management/org_settings/service_accounts.md:83]:
```suggestion
When creating a service account, you can give it any subset of the roles and permissions that you have. The exception is if you have the User Access Manage permission, which effectively gives you administrator access to do anything in Datadog. Datadog accounts with the User Access Manage permission have no restrictions on the roles and permissions they can assign to service accounts.
```

File: content/en/account_management/org_settings/service_accounts.md [hunk 13]
```diff
@@ -0,0 +1,94 @@
...
+
+
+## Notifications
+
+Datadog sends a notification to the email address associated with the service account when the following actions occur:
+- Create an application key
+- Revoke an application key
+- Disable the service account
+
+[1]: /account_management/org_settings/
```
Comment (alai97) [inline content/en/account_management/org_settings/service_accounts.md:93]:
```suggestion

## Further reading

{{< partial name="whats-next/whats-next.html" >}}

[1]: /account_management/org_settings/
```

File: content/en/account_management/org_settings.md [hunk 14]
```diff
@@ -82,10 +82,11 @@ To rename your organization, click the **Edit** button in the **Preferences** ta
 
 [1]: /account_management/users/default_roles/
```
Comment (alai97) [inline content/en/account_management/org_settings.md:83]:
```suggestion

## Further reading

{{< partial name="whats-next/whats-next.html" >}}

[1]: /account_management/users/default_roles/
```

**Review by alai97 — CHANGES_REQUESTED** [review]
Some copy suggestions. Can you add some Further Reading links to the front matter so the Further Reading partial contains more doc links?

**Comment by urseberry** [comment]
This is ready for re-review.

**Review by alai97 — APPROVED** [review]
LGTM!

**Pull request merged**